Userland/Demos/Starfield/Starfield.cpp

```diff
@@ -148,6 +171,9 @@ int main(int argc, char** argv)
     if (refresh_rate == 0)
         refresh_rate = 16;
 
+    if (speed == 0)
```

Comment:
Is this necessary? Can we not just keep the default at 1, I don't think argsparser would override it if the option is not given?

Reply:
A nicer scheme would be to make a custom Option, and reject the bad value right from the start:
```c++
    args_parser.add_option(Core::ArgsParser::Option {
        .requires_argument = true,
        .help_string = "Speed (default = 1)",
        .long_name = "speed",
        .short_name = 's',
        .value_name = "speed",
        .accept_value = [&speed](char const* s) {
            auto maybe_value = StringView(s).to_uint();
            if (!maybe_value.has_value())
                return false;
            if (*maybe_value < 1)
                return false;
            speed = *maybe_value;
            return true;
        },
    });
```